packages/client/src/views/SysAdmin/Team/user/UserList.tsx

```diff
@@ -601,6 +678,39 @@ function UserListComponent(props: IProps) {
                 }
               ]}
             />
+            <ResponsiveModal
+              id="user-reset-password-modal"
+              show={toggleResetPassword.modalVisible}
+              handleClose={() => toggleUserResetPasswordModal()}
+              title={intl.formatMessage(messages.resetUserPasswordModalTitle)}
+              actions={[
+                <TertiaryButton
+                  id="reset-password-cancel"
+                  key="reset-password-cancel"
+                  onClick={() => toggleUserResetPasswordModal()}
+                >
+                  {intl.formatMessage(buttonMessages.cancel)}
+                </TertiaryButton>,
+                <PrimaryButton
+                  id="reset-password-send"
+                  key="reset-password-send"
+                  onClick={() => {
+                    if (toggleResetPassword.selectedUser?.id) {
+                      resetPassword(toggleResetPassword.selectedUser.id)
+                    }
+                    toggleUserResetPasswordModal()
+                  }}
+                >
+                  {intl.formatMessage(buttonMessages.send)}
+                </PrimaryButton>
```

Comment:
if possible I would use the `<Button type="tertiary">` & `<Button type="primary">` here